### ISSUE TITLE
Gateway Readme: errors building OpenZWave

### ISSUE THREAD

**Issue by cyrfer** [mozilla-iot/gateway#965]
maybe related to #964 
```
$ sudo ln -s /usr/include/libusb-1.0/libusb.h /usr/include/libudev.h

pi@raspberrypi:~/software/open-zwave $ ls -l /usr/include/libudev.h 
lrwxrwxrwx 1 root root 32 Apr 29 08:27 /usr/include/libudev.h -> /usr/include/libusb-1.0/libusb.h

pi@raspberrypi:~/software/open-zwave $ CFLAGS=-D_GLIBCXX_USE_CXX11_ABI=0 make && sudo CFLAGS=-D_GLIBCXX_USE_CXX11_ABI=0 make install
LDFLAGS="" CPPFLAGS="" make -C /home/pi/software/open-zwave/cpp/build/ -
make[1]: Entering directory '/home/pi/software/open-zwave/cpp/build'
Building OpenZWave Version 1.4-3016-gc8441b10-dirty
Building hid.o
/home/pi/software/open-zwave/cpp/hidapi/linux/hid.c:116:41: error: ‘struct udev_device’ declared inside parameter list will not be visible outside of this definition or declaration [-Werror]
 static wchar_t *copy_udev_string(struct udev_device *dev, const char *udev_name)
                                         ^~~~~~~~~~~
/home/pi/software/open-zwave/cpp/hidapi/linux/hid.c: In function ‘copy_udev_string’:
/home/pi/software/open-zwave/cpp/hidapi/linux/hid.c:118:25: error: implicit declaration of function ‘udev_device_get_sysattr_value’ [-Werror=implicit-function-declaration]
  return utf8_to_wchar_t(udev_device_get_sysattr_value(dev, udev_name));
                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/pi/software/open-zwave/cpp/hidapi/linux/hid.c:118:25: error: passing argument 1 of ‘utf8_to_wchar_t’ makes pointer from integer without a cast [-Werror=int-conversion]
/home/pi/software/open-zwave/cpp/hidapi/linux/hid.c:97:17: note: expected ‘const char *’ but argument is of type ‘int’
 static wchar_t *utf8_to_wchar_t(const char *utf8)
                 ^~~~~~~~~~~~~~~
/home/pi/software/open-zwave/cpp/hidapi/linux/hid.c: In function ‘get_device_string’:
/home/pi/software/open-zwave/cpp/hidapi/linux/hid.c:251:9: error: implicit declaration of function ‘udev_new’ [-Werror=implicit-function-declaration]
  udev = udev_new();
         ^~~~~~~~
```

**Comment by rzr** [comment]
Maybe building a debian package for raspbian would sort out all those link hacks, I can help if needed

**Comment by mrstegeman** [comment]
This is totally invalid:
```
$ sudo ln -s /usr/include/libusb-1.0/libusb.h /usr/include/libudev.h
```

`libusb` != `libudev`

The README has been updated to fix #964.
